lti_consumer/plugin/views.py

```diff
@@ -175,6 +177,38 @@ def deep_linking_response_endpoint(request, lti_config_id=None):
         return HttpResponse(status=403)
 
 
+@require_http_methods(['GET'])
+def deep_linking_content_endpoint(request, lti_config_id=None):
+    """
+    Deep Linking endpoint for rendering Deep Linking Content Items.
+    """
+    try:
+        # Get LTI Configuration
+        lti_config = LtiConfiguration.objects.get(id=lti_config_id)
+
+        # This checks if user can perform LTI launch. If yes then user can also view LTI-DL content
+        allowed = has_access(request.user, 'load', lti_config.block, lti_config.location.course_key)
+        if not allowed:
+            return HttpResponse(status=403)
+
+        # Get all LTI-DL contents
+        content_items = LtiDlContentItem.objects.filter(lti_configuration=lti_config)
+
+        # If no LTI-DL contents found for current configuration, throw 404 error
+        if content_items.count() == 0:
```

Comment:
I feel doing a exists will be better than checking count 
https://stackoverflow.com/questions/6868939/in-django-what-is-the-most-efficient-way-to-check-for-an-empty-query-set

Reply:
Yup, ``exists`` would be sufficient in this case.